api/src/main/java/org/openmrs/module/interop/InteropConstant.java

```diff
@@ -16,4 +16,10 @@ public class InteropConstant {
 	public static final String GP_ENABLE_KAFKA = "interop.enableKafka";
 	
 	public static final String GP_ENABLE_OPENHIM = "interop.enableOpenHIM";
+	
+	public static final String NATIONAL_UNIQUE_PATIENT_NUMBER_UUID = "f85081e2-b4be-4e48-b3a4-7994b69bb101";
+	
+	public static final String MFL_LOCATION_ATTRIBUTE_UUID = "8a845a89-6aa5-4111-81d3-0af31c45c002";
+	
+	public static final String INTEROP_PROVIDER_UUID = "185e3e96a44-fced3e76-6b4f-45bd-abb2-ddafbfbc9aa9";
```

Comment:
What role do these variables play? I suggest we focus on making the module agnostic of the openmrs flavor it's running on. We might consider abstracting this to an extension of the interop module.